packages/fxa-profile-server/lib/routes/ecosystem_anon_id/post.js

```diff
@@ -6,6 +6,59 @@ const Joi = require('@hapi/joi');
 
 const logger = require('../../logging')('routes.ecosystem_anon_id.post');
 const notifyProfileUpdated = require('../../updates-queue');
+const AppError = require('../../error');
+const config = require('../../config');
+const request = require('../../request');
+
+const AUTH_SERVER_URL =
+  config.get('authServer.url') + '/account/metrics/ecosystemAnonId';
+
+const updateAuthServer = function (credentials, anonId) {
+  return new Promise((resolve, reject) => {
+    request.get(
+      AUTH_SERVER_URL,
+      {
+        headers: {
+          Authorization: 'Bearer ' + credentials.token,
+        },
+        body: {
+          anonId,
+        },
+        json: true,
+      },
+      (err, res, body) => {
```

Comment:
I pulled all of this error handling from https://github.com/mozilla/fxa/blob/main/packages/fxa-profile-server/lib/routes/_core_profile.js -- I can remove if it feels like overkill.